e2e/various-suite/navigation.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { e2e } from '../utils';
+import { fromBaseUrl } from '../utils/support/url';
+
+function megaMenuShouldBeDocked() {
+  // TODO: is this a good way to assert the menu is docked?
```

Comment:
Not sure whether it's possible to get the status from there but it's also available in [local storage](https://github.com/grafana/grafana/blob/afa697f95449bdedb919572301470cd27f9ec89a/public/app/core/components/AppChrome/AppChromeService.tsx#L37).

Reply:
honestly, i think it's enough just to check for visibility of `navbarmenu`. the menu being consistently visible is the important bit

Reply:
Great catch. i initially didn't see that in the DOM, but I think that's a better node to assert on.

Reply:
might be worth putting that in the e2e-selectors now? up2u 🤔